modules/objdetect/test/test_qrcode.cpp

```diff
@@ -33,6 +34,8 @@ std::string qrcode_images_multiple[] = {
   "5_qrcodes.png", "6_qrcodes.png", "7_qrcodes.png", "8_close_qrcodes.png"
 };
 
+static std::set<std::pair<std::string, QRDetectMethod>> dropped_samples = {{"5_qrcodes.png", QRDetectMethod::ARUCO}};
```

Comment:
`disabled_samples` is more comprehensive.

Reply:
fixed